src/forms/NewStudyForm.js

```diff
@@ -1,351 +1,453 @@
-                                )[0].text
-                              }
-                            </List.Item>
-                          ))}
-                        </List.List>
+        SUBMIT
+      </Button>
+      <Confirm
+        open={confirmOpen}
+        onCancel={() => setConfirmOpen(false)}
+        onConfirm={handleSubmit}
+        header="Create Study"
+        content={
+          <Container as={Segment} basic padded>
+            <p>The following resources will be created for this study</p>
+            <List bulleted>
+              <List.Item>Dataservice study</List.Item>
+              <List.Item>S3 bucket</List.Item>
+              <List.Item>Cavatica delivery project</List.Item>
+              {workflowType.length > 0 && (
+                <List.Item>
+                  Cavatica harmonization projects
+                  <List.List>
+                    {workflowType.map(type => (
+                      <List.Item key={type}>
+                        {
+                          workflowOptions.filter(obj => obj.value === type)[0]
+                            .text
+                        }
                       </List.Item>
-                    )}
-                  </List>
-                </Container>
-              }
-              confirmButton={
-                <Button
-                  primary
-                  floated="right"
-                  type="submit"
-                  loading={isSubmitting}
-                >
-                  SUBMIT
-                </Button>
-              }
+                    ))}
+                  </List.List>
+                </List.Item>
+              )}
+            </List>
+          </Container>
+        }
+        confirmButton={
+          <Button primary floated="right" type="submit" loading={isSubmitting}>
+            SUBMIT
+          </Button>
+        }
+      />
+    </>
+  );
+};
+
+const NewStudyForm = ({submitValue, apiErrors}) => {
+  const [confirmOpen, setConfirmOpen] = useState(false);
+  const [workflowType, setSelection] = useState([
+    'bwa_mem',
+    'gatk_haplotypecaller',
+  ]);
+  const [focused, setFocused] = useState('');
+  const [activeStep, setActiveStep] = useState('info');
+  return (
+    <Formik
+      initialValues={{
+        externalId: '',
+        name: '',
+        shortName: '',
+        description: '',
+        releaseDate: '',
+        version: '',
+        attribution: '',
+        anticipatedSamples: 0,
+        awardeeOrganization: '',
+      }}
+      validate={values => {
+        let errors = {};
+        if (!values.externalId) {
+          errors.externalId = 'Required';
+        }
+        if (!values.name) {
+          errors.name = 'Required';
+        }
+        if (values.anticipatedSamples < 0) {
+          errors.anticipatedSamples = 'Invalid number of anticipated samples';
+        }
+        if (values.version.length > 10) {
+          errors.version = 'Maximum 10 characters long';
```

Comment:
maybe a message along the lines of `Max. 10 characters, typically in the format v[0-9].p[0-9], `